server/src/Application/DependencyInjection.cs

```diff
@@ -11,6 +15,18 @@ public static IServiceCollection AddApplication(this IServiceCollection services
         services.AddScoped<ILectureService, LectureService>();
         services.AddScoped<IFlashcardService, FlashcardService>();
         services.AddScoped<IFlashcardSetService, FlashcardSetService>();
+        services.AddScoped<IValidator<CreateCourseDto>, CreateCourseRequestValidator>();
```

Comment:
All validators can be registered automatically: https://stackoverflow.com/questions/63023438/fluentvalidation-how-to-register-all-validators-automatically-from-another-asse